Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_expedition.yml

```diff
@@ -107,6 +107,22 @@
     - id: SpeedLoaderMagnum
       amount: 2
 
+- type: entity
+  parent: WeaponCaseShort
+  id: WeaponCaseShortTheArbitrator
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponRevolverarbitrator
```

Comment:
Should be a different prototype, the one that has contraband component attached to it.

Reply:
why would it be contraband?

Reply:
Because if you find a gun on expedition it's "considered unregistered", as in not obtained through official means. Mechanically this is achieved through creating a separate entity. If you need example of how it's done, look in: `\Resources\Prototypes\_NF\Entities\Objects\Weapons\Guns\expedition_guns.yml`

Reply:
I did it